lepton/aws.go

```diff
@@ -472,7 +472,15 @@ func (p *AWS) CreateSG(ctx *Context, svc *ec2.EC2, imgName string) (string, erro
 	if len(result.Vpcs) == 0 {
 		fmt.Println("No VPCs found to associate security group with.")
 	}
-	vpcID := aws.StringValue(result.Vpcs[0].VpcId)
+
+	var vpcID string
+	//TODO: This will fail if there is no default VPC. Need to implement feature where user should be able to mention VPC
+	for i, s := range result.Vpcs {
+		isDefault := *s.IsDefault
```

Comment:
maybe just default to the first one if no default? although iirc that change went into affect something like 7 years ago; can add an issue later to specify vpc to place in

Reply:
The "runinstanceinput" needs VPC if we don't specify one it goes back to default. 

* [EC2-VPC] If you don't specify a subnet ID, we choose a default subnet
from your default VPC for you. If you don't have a default VPC, you must
specify a subnet ID in the request.* [link](https://docs.aws.amazon.com/sdk-for-go/v2/api/service/ec2/#Client.RunInstancesRequest). Chossing first one will result in same problem we are trying to fix.